pkg/kubelet/prober/prober.go

```diff
@@ -129,6 +133,9 @@ func (pb *prober) probe(probeType probeType, pod *v1.Pod, status v1.PodStatus, c
 	} else {
 		klog.V(3).InfoS("Probe succeeded", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
 	}
+	if !klog.V(3).Enabled() {
+		pb.trace.LogWithFrequency(fmt.Sprintf("%s-%s-%s", probeType, klog.KObj(pod), container.Name), "Probe succeeded", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "output", output)
```

Comment:
I wonder if this print is reasonable?
Is the information in `"%s-%s-%s", probeType, klog.KObj(pod), container.Name` repeated with the following?